packages/cli/src/cache.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ *
+ */
+import fs from 'fs';
+import path from 'path';
+import crypto from 'crypto';
+// $FlowFixMe
+import { mkdirp } from 'mkdirp';
+
+// Default cache directory in `node_modules/.stylex-cache`
+export function getDefaultCachePath() {
+  return process.env.NODE_ENV === 'test'
+    ? path.join('node_modules', '.stylex-cache-test')
+    : path.join('node_modules', '.stylex-cache');
+}
+
+function getCacheFilePath(cachePath, filePath) {
+  const fileName = filePath.replace(/[\\/]/g, '__');
+  return path.join(cachePath, `${fileName}.json`);
+}
+
+export function readCache(filePath) {
+  const cacheFile = getCacheFilePath(getDefaultCachePath(), filePath);
```

Comment:
Do you have a way to override `getDefaultCachePath` already? If not, please add some comments that the "defaultCachePath` is currently the only cache path.

Reply:
adding this in a fast follow!